devfiles/php-laravel/devfile.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: php-laravel
+projects:
+-
+  name: laravel-realworld-example-app
+  source:
+    type: git
+    location: "https://github.com/gothinkster/laravel-realworld-example-app.git"
+components:
+-
+  type: chePlugin
+  id: redhat/php/latest
+  memoryLimit: 1000Mi
```

Comment:
it's not major comment at all, but it could be `1Gi`

Reply:
changed to `1Gi`